### ISSUE TITLE
Verbose splash screen

### ISSUE THREAD

**Issue by adocampo** [sddm/sddm#920]
I'm trying to customize SDDM to be more verbose so what the system is doing. I hope you can shred some light.

The moving icon when loading desktop doesn't seems to be enough for my client, and with MATE, we developed a zenity script which showed what the system was doing while it was loading the desktop.

Chakra theme heritage shows a loading screen which inspired me to do something in there. It shows the full date by default, which I changed by a "Loading desktop, please wait..." which seems ok by the moment, but the final client wants to show more information, like "Mounting filesystem X"|"Installing software Y"|"Mapping printer Z" using the same logic we did on the zenity script.

On the old environment, with lightdm and MATE:
When a users logged in, pam.d-common-session launched a bash script calling a python script which mounts shares, installing apps, mapping printers, etc and that python writed all the output to a /tmp/output file. Then, a bash script is launched tailing that /tmp/output and showing it via zenity until the desktop was shown (using wmctrl).
That gave the user all the information the system was doing, besides a progress bar.

On the new environment with sddm and Plasma.
Basically what I need is to know how can I tail that /tmp/ouput file and show it via sddm through /usr/share/plasma/look-and-feel/org.kde.heritage.desktop/contents/splash/Splash.qml
I'm horribly new to Qt Quick (just installing right now Qt Creator for the first time) and I just like to know if that what I want is even possible and if so, if there are some example how can I read text from a file.

Thank you
